test/Microsoft.Azure.WebJobs.Host.UnitTests/Common/TriggerTests.cs

```diff
@@ -214,15 +233,83 @@ public void TestSingleStringDispatch()
             Assert.Equal(e0.Message, items[0]);
         }
 
+        static void AddItem2ByteArrayConverter(IConverterManager cm)
+        {
+            cm.AddConverter<FakeQueueData, byte[]>(msg => System.Text.Encoding.UTF8.GetBytes(msg.Message));
+        }
+        static void AddItem2ByteConverter(IConverterManager cm)
+        {
+            cm.AddConverter<FakeQueueData, byte>(msg => msg.Byte);
+        }
+
+        [Fact]
+        public void TestByteArrayDispatch()
+        {
+            var e0 = new FakeQueueData
+            {
+                Message = "ABC"
+            };
+
+            var client = new FakeQueueClient();
+            client.SetConverters = AddItem2ByteArrayConverter;
+            
+            var items = Run<FunctionsByteArray>(client, e0);
+            Assert.Equal(1, items.Length);
+            
+            // This uses the Item --> byte[] converter. Dispatch as a single item.
+            // Received as 1 object, a byte[]. 
+            var bytes = System.Text.Encoding.UTF8.GetBytes(e0.Message);
+            Assert.Equal(bytes, items[0]);
+        }
+                
+        [Fact]
+        public void TestByteArrayDispatch2()
```

Comment:
can you please give this test and the one below it more descriptive names so it is clear what they're verifying?